unit_test/fb_serializer_test.cc

```diff
@@ -94,11 +94,14 @@ BOOST_AUTO_TEST_CASE(fb_serializer_ranking_event) {
     BOOST_CHECK_EQUAL_COLLECTIONS(event.probabilities()->begin(), event.probabilities()->end(), expected_prob.begin(), expected_prob.end());
     BOOST_CHECK_EQUAL(event.deferred_action(), false);
     BOOST_CHECK_EQUAL(event.pass_probability(), 0.33f);
-    if (i % 2 == 0) {
```

Comment:
These module operators are no longer appropriate. A switch may be better suited. Also, think about moving away from this hardcoded 3 and referencing the size of the enum.